lib/ansible/modules/network/ironware/ironware_vlan.py

```diff
@@ -75,15 +76,6 @@
     aggregate:
       - vlan_id: 4000
       - {vlan_id: 4001, name: vlan-4001}
-
-- name: Create vlans with a mix of tagged and untagged ports
```

Comment:
This example resulted in some unpredictable behavior so it has been removed. Suggest vlan could be created then changed to tagged/untagged using the l2interface module...